web/locale/en_US/messages.po

```diff
@@ -3157,3 +3157,30 @@ msgstr "Change Role"
 
 msgid "I have read and accept the <a ui-sref=\"terms-and-conditions\" target=\"_blank\"><b>Terms and Conditions here</b></a>"
 msgstr "I have read and accept the <a ui-sref=\"terms-and-conditions\" target=\"_blank\"><b>Terms and Conditions here</b></a>"
+
+msgid "Donate"
+msgstr "Donate"
+
+msgid "Maintained by"
+msgstr "Maintained by"
+
+msgid "Attend a Dojo"
+msgstr "Attend a Dojo"
+
+msgid "Volunteer"
```

Comment:
they are in the Header and Footer,  you can see in the issue #934 @niccokunzmann had a picture showing that these don't translate, so I have added these in messages.po file, so that these can be translated, just how I did for "I have read and accepted the Terms and Conditions."